packages/react-core/src/components/NotificationDrawer/NotificationDrawerHeader.tsx

```diff
@@ -30,7 +30,9 @@ export const NotificationDrawerHeader: React.FunctionComponent<NotificationDrawe
     <Text component={TextVariants.h1} className={css(styles.notificationDrawerHeaderTitle)}>
       {title}
     </Text>
-    {count && <span className={css(styles.notificationDrawerHeaderStatus)}>{`${count} ${unreadText}`}</span>}
+    {count !== undefined && (
+      <span className={css(styles.notificationDrawerHeaderStatus)}>{`${count} ${unreadText}`}</span>
```

Comment:
this seems a bit too rigid of a format and won't work well with i18n

Reply:
What do you propose?

Reply:
Not a blocker if this was already released, but I question if the count prop here is needed, user could just pass `<NotificationDrawerHeader unreadText=`${count} unread` />`

Reply:
ah
`NotificationDrawerGroup` also expects a `count: number;` to display in its header.

I guess I'm not sure I see a big difference, and it would be a breaking change to remove `count` from `NotificaitonDrawerHeader`

Reply:
It's a little different in the drawer group because it's just the count there. Here we mandate that count is followed by a space and then a string which is not correct for some languages. This already existed before so not a blocker for this PR tho! But we are accruing technical debt here, since this is a beta component I am not sure if breaking changes are allowed or not.

Reply:
https://github.com/patternfly/patternfly-react/issues/4651